PhysicsTools/MVAComputer/interface/BitSet.h

```diff
@@ -130,21 +131,17 @@ namespace PhysicsTools {
 
     BitSet(const BitSet &orig) : bits_(orig.bits_) {
       std::size_t words = bitsToWords(bits_);
-      if (words) {
-        store = new Word_t[words];
-        std::memcpy(store, orig.store, words * sizeof(Word_t));
-      } else
-        store = nullptr;
+      assert(words);
+      store = new Word_t[words];
+      std::memcpy(store, orig.store, words * sizeof(Word_t));
     }
 
     /// construct BitSet with a fixed size of \a bits bits
     BitSet(size_t bits) : bits_(bits) {
       std::size_t words = bitsToWords(bits);
-      if (words) {
-        store = new Word_t[words];
-        std::memset(store, 0, sizeof(Word_t) * words);
-      } else
-        store = nullptr;
+      assert(words);
+      store = new Word_t[words];
+      std::memset(store, 0, sizeof(Word_t) * words);
```

Comment:
I assume that `bits_ > 0` and in that case `bitsToWords(bits_)` should always return `words>0`